multi_token_support_program/src/main.leo

```diff
@@ -72,7 +72,7 @@ program multi_token_support_program.aleo {
       decimals: 6u8,
       supply: 0u128, // placeholder value; not used
       max_supply: 0u128, // placeholder value; not used
```

Comment:
we need to set a value for this now that we're using `mint` via the new credits wrapper

Reply:
10B, I think should be enough